SPECS/xen.spec

```diff
@@ -110,355 +110,377 @@ Patch77: backport-e3eb84e33c36.patch
+Patch380: 0002-x86-spec-ctrl-Mitigations-for-LazyFPU.patch
+Patch381: xsa264-4.10.patch
+Patch382: xsa265-4.7.patch
+Patch383: backport-55674ed8c826.patch
+Patch384: backport-839826b094a0.patch
+Patch385: backport-e7956461f76f.patch
+Patch386: backport-bd63f041923a.patch
+Patch387: backport-087369973214.patch
+Patch388: backport-dc111e9f0d99.patch
+Patch389: backport-a4041364c3ae.patch
+Patch390: backport-9858a1f3fb9a.patch
+Patch391: backport-c0e854be515b.patch
+Patch392: backport-e90e2431a4ae.patch
+Patch393: backport-97aff087fd0b.patch
+Patch394: backport-fa807e2ff69d.patch
+Patch395: backport-bce2dd64b52e.patch
+Patch396: backport-91ca84c862b1.patch
+Patch397: backport-f30e3cf34042.patch
+Patch398: backport-730dc8d2c9e1.patch
+Patch399: backport-f54b63e8617a.patch
+Patch400: backport-94fda356fcdc.patch
+Patch401: backport-4d94828cf111.patch
+Patch402: backport-80599f0b7701.patch
+Patch403: backport-be73a842e642.patch
+Patch404: backport-ee7689b94ac7.patch
+Patch405: backport-1ac46b556326.patch
+Patch406: xsa269-4.8.patch
+Patch407: 0001-x86-spec-ctrl-Yet-more-fixes-for-xpti-parsing.patch
+Patch408: 0002-x86-spec-ctrl-Calculate-safe-PTE-addresses-for-L1TF-.patch
+Patch409: 0003-x86-spec-ctrl-Introduce-an-option-to-control-L1TF-mi.patch
+Patch410: 0004-x86-shadow-Infrastructure-to-force-a-PV-guest-into-s.patch
+Patch411: 0005-x86-mm-Plumbing-to-allow-any-PTE-update-to-fail-with.patch
+Patch412: 0006-x86-pv-Force-a-guest-into-shadow-mode-when-it-writes.patch
+Patch413: 0007-x86-spec-ctrl-CPUID-MSR-definitions-for-L1D_FLUSH.patch
+Patch414: 0008-x86-msr-Virtualise-MSR_FLUSH_CMD-for-guests.patch
+Patch415: 0009-x86-spec-ctrl-Introduce-an-option-to-control-L1D_FLU.patch
+Patch416: xsa278-4.7.patch
+Patch417: xsa275-4.7-1.patch
+Patch418: xsa275-4.7-2.patch
+Patch419: xsa280-4.9-1.patch
+Patch420: xsa280-4.7-2.patch
+Patch421: xsa282-4.9-1.patch
+Patch422: xsa282-4.8-2.patch
+Patch423: 0001-VMX-allow-migration-of-guests-with-SSBD-enabled.patch
+Patch424: xsa283.patch
+Patch425: xsa284.patch
+Patch426: xsa285.patch
+Patch427: xsa287-4.7.patch
+Patch428: xsa288-4.7.patch
+Patch429: xsa290-4.7-1.patch
+Patch430: xsa290-4.7-2.patch
+Patch431: xsa293-4.7-1.patch
+Patch432: xsa293-4.7-2.patch
+Patch433: xsa293-4.7-3.patch
+Patch434: backport-ff9b9d540f1b.patch
+Patch435: backport-fc46e159a6b1.patch
+Patch436: backport-ffb60a58df48.patch
+Patch437: 0001-x86-spec-ctrl-Reposition-the-XPTI-command-line-parsi.patch
+Patch438: 0002-x86-msr-Definitions-for-MSR_INTEL_CORE_THREAD_COUNT.patch
+Patch439: 0003-x86-boot-Detect-the-firmware-SMT-setting-correctly-o.patch
+Patch440: xsa297-4.7-1.patch
+Patch441: xsa297-4.7-2.patch
+Patch442: xsa297-4.7-3.patch
+Patch443: xsa297-4.7-4.patch
+Patch444: 0001-xen-Introduce-vcpu_sleep_nosync_locked.patch
+Patch445: 0002-xen-schedule-Fix-races-in-vcpu-migration.patch
+
+Provides: gitsha(ssh://git@code.citrite.net/xsu/xen.git) = 280a5568939c4a5832be787c8e0c23a19f30935a
```

Comment:
We don't want all those useless provides from XenServer. I kept them in 7.6 and 8.0 for easier merges, but here we only want to apply the patches, not to backport this nonsense from 7.6. Comment applies to all added `Provides:`

Now that I think of it, I should have updated my SRPM import script to remove them all in 8.0.

Reply:
Done